executor/show.go

```diff
@@ -1326,10 +1324,17 @@ func ConstructResultOfShowCreateDatabase(ctx sessionctx.Context, dbInfo *model.D
 			fmt.Fprintf(buf, "COLLATE %s ", dbInfo.Collate)
 		}
 		fmt.Fprint(buf, "*/")
-		return nil
 	}
 	// MySQL 5.7 always show the charset info but TiDB may ignore it, which makes a slight difference. We keep this
 	// behavior unchanged because it is trivial enough.
+	if dbInfo.DirectPlacementOpts != nil {
+		fmt.Fprintf(buf, " %s", dbInfo.DirectPlacementOpts)
+	}
+	if dbInfo.PlacementPolicyRef != nil {
+		fmt.Fprintf(buf, " PLACEMENT POLICY = '%s'", dbInfo.PlacementPolicyRef.Name.O)
+		// FIXME: Support properly quoted policy names in the parser!
+		//fmt.Fprintf(buf, " PLACEMENT POLICY = %s", stringutil.Escape(dbInfo.PlacementPolicyRef.Name.O, sqlMode))
```

Comment:
I believe this is fixed in https://github.com/pingcap/parser/pull/1334 ?

Reply:
Yes, I removed it. It will be fixed in tidb by https://github.com/pingcap/tidb/pull/28055